org.eclipse.xtend.lib/build.gradle

```diff
@@ -6,3 +6,23 @@ dependencies {
 	compile project(':org.eclipse.xtext.xbase.lib')
 	compile project(':org.eclipse.xtend.lib.macro')
 }
+
+task copyGwtSources(dependsOn: 'build') {
```

Comment:
@oehme 

yes but i googled that and i did not find any example that does that.

so 

(1) how do i do a source jar config. respectively how do a 

dependencies {
gtwCrap project(':org.eclipse.xtext.xbase.lib', classifier: source)
}